### PR TITLE
Update header

### PR DESCRIPTION
**Pull request by nickbristow**
Update header to match mock ups.
### JIRA issue link
https://agile6.atlassian.net/browse/DM-888
## Description - what does this code do?
Update header to match mock ups

## Screenshots, Gifs, Videos from application (if applicable)
Diffusion_Marketplace

Diffusion_Marketplace

## Link to mock-ups/mock ups (image file if you have it) (if applicable)
https://www.figma.com/file/DDUSxYiD1MaWNrGFmMjY8d8a/Diffusion-Marketplace-Designs?node-id=767%3A1

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Unit tests written (if applicable)
- [ ] e2e/accessibility tests written (if applicable)
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating JIRA issue
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -55,6 +55,7 @@ group :development, :test do
   gem 'rspec_junit_formatter'
   gem 'simplecov'
   gem 'shoulda-matchers', require: false
+  gem 'dotenv-rails'
```
Comment (ahay-agile6) [inline Gemfile]: we use figaro for this. is it not working for you?

File: app/assets/stylesheets/dm/custom_font.scss [hunk 2]
```diff
@@ -24,3 +24,7 @@ i.dm {
 
   &.dm-saluting-veteran:before { content: "\e900";}
 }
+
+.font-normal {
+  font-weight: normal;
```
Comment (ahay-agile6) [inline app/assets/stylesheets/dm/custom_font.scss]: you can probably use `.text-normal` from uswds: https://designsystem.digital.gov/utilities/text-styles/

File: app/assets/images/empty-profile-header.svg [hunk 3]
```diff
@@ -0,0 +1,5 @@
+<svg width="40" height="40" viewBox="0 0 40 40" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (ahay-agile6) [inline app/assets/images/empty-profile-header.svg]: can we use font awesome for this? is it a different icon?

File: Gemfile (hunk 1 above)
Comment (nickbristow) [reply Gemfile]: err...yes. I forgot we were using that.

File: app/assets/stylesheets/dm/custom_font.scss (hunk 2 above)
Comment (nickbristow) [reply app/assets/stylesheets/dm/custom_font.scss]: 👍

File: app/assets/images/empty-profile-header.svg (hunk 3 above)
Comment (nickbristow) [reply app/assets/images/empty-profile-header.svg]: I asked becca for the icon, looks like it is just the font awesome one. I've updated this.

**Pull request merged**